### PR TITLE
feat: :sparkles: Adds keys state to allow toggle keys

### PR DESCRIPTION
**Pull request by IsmaelEzequiel**
Hi, adds active state to buttons #25

### CODE REVIEW

**Review by willianrod — CHANGES_REQUESTED** [review]
We could reuse the same style already applied when dragging over a key

File: src/renderer/components/key/index.tsx [hunk 1]
```diff
@@ -75,10 +76,15 @@ const Key: React.FC<IKey> = ({ button }) => {
       tabIndex={-1}
     >
       <img
-        className={styles.buttonOverlay}
+        className={
+          currentKey?.position === button.id
+            ? styles.buttonOverlayActive
+            : styles.buttonOverlay
+        }
```
[the remark this answers is not in the document]
Comment (IsmaelEzequiel) [reply src/renderer/components/key/index.tsx:83]: Sure, I'll make the changes

**Comment by IsmaelEzequiel** [comment]
@willianrod fixed!

**Review by willianrod — APPROVED** [review]
LGTM!

**Comment by willianrod** [comment]
@IsmaelEzequiel thanks for your contribution. Hope to see you again ;)

**Pull request merged**